### DISCUSSION TITLE
Motherboard jumper settings, COASt size, etc.

### DISCUSSION THREAD

**Discussion by shitsux420** [86Box/86Box#2912] [Ideas]
**Is your feature request related to a problem? Please describe.**
I guess missing a wanted feature could be a problem?

**Describe the solution you'd like**
I'd like to request emulation of motherboard jumpers on the various 386/486 and Pentium boards. This would mainly be useful for setting the FSB, processor clock and processor voltage jumpers to over- and underclock the CPU. Of course this could be implemented as a simple selection for these items at the machine section of the configuration menu via drop down menu.

Another feature I'd like to see added is configurable COASt module size. Right now (v. 3.7.1) the external L2 cache for Pentiums is hardlocked at 256kb, contemporary benchmarks of this era have shown that having 512k or no cache at all could result in better and worse performance respectfully and thus this could be reflected into the emulated machine as well.

**Describe alternatives you've considered**
Just not being so anal about things.

**Additional context**
I have no idea how hard these features would be to implement nor what kind of documentation would be needed other than motherboard manuals that go over the jumpers for request #1.

**Comment by OBattler** [comment]
The hardest part would be all the moderators that we have little to no information about.

**Comment by OBattler** [comment]
I meant all the motherboards, not moderators.

**Comment by shitsux420** [comment]
I understand. Hopefully these would someday be added to increase the versatility of the emulation.

By the way, another feature I'd like to see is the support for SDRAM. Currently every Intel-chipset board reports only EDO RAM in the BIOS even though the P2 era 440BX boards support SDRAM. Some boards like ASUS P2B-LS only support SDRAM, so I guess this could technically be considered a bug as well. Of course this need not be "real" support for SDRAM, just that the emulator reports the correct RAM expected by the motherboard.

**Comment by OBattler** [comment]
We do emulate SDRAM, but the reporting as EDO is not fixable - it would need the SDRAM test mode implemented, which would be potentially detrimental to performance.
